src/components/FindWallet/WalletFilterSidebar.tsx

```diff
@@ -0,0 +1,496 @@
+          description:
+            "User is able to buy crypto with fiat directly in the wallet.",
+          filterKey: "buy_crypto",
+          showOptions: undefined,
+          options: [],
+        },
+        {
+          title: "Withdraw crypto",
+          icon: <WithdrawCrypto />,
+          description:
+            "User is able to withdraw to fiat directly in the wallet.",
+          filterKey: "withdraw_crypto",
+          showOptions: undefined,
+          options: [],
+        },
+      ],
+    },
+    {
+      title: "Smart contracts",
+      open: true,
+      items: [
+        {
+          title: "Multisig",
+          icon: <Multisig />,
+          description:
+            "Wallets that require two or more signatures from private keys for a transaction.",
+          filterKey: "multisig",
+          showOptions: undefined,
+          options: [],
+        },
+        {
+          title: "Social recovery",
+          icon: <SocialRecover />,
+          description:
+            "Wallets that allow guardians to change the signing key for smart contract wallets.",
+          filterKey: "social_recovery",
+          showOptions: undefined,
+          options: [],
+        },
+      ],
+    },
+  ])
+
+  const setOpen = (idx) => {
+    const updatedFilterOptions = [...filterOptions]
+    updatedFilterOptions[idx].open = !updatedFilterOptions[idx].open
+    setFilterOptions(updatedFilterOptions)
+  }
+
+  const setShowOptions = (idx, itemidx, value) => {
+    const updatedFilterOptions = [...filterOptions]
+    updatedFilterOptions[idx].items[itemidx].showOptions =
+      !updatedFilterOptions[idx].items[itemidx].showOptions
+    setFilterOptions(updatedFilterOptions)
+
+    const keys = updatedFilterOptions[idx].items[itemidx].options.map(
+      (item) => item.filterKey
+    )
+    updateFilterOptions(keys, value)
+  }
+
+  return (
+    <Container>
+      {filterOptions.map((filterOption, idx) => {
+        return (
+          <FilterPanel>
+            <Header
+              onClick={() => {
+                setOpen(idx)
+              }}
+            >
+              <h3>{filterOption.title}</h3>
+              <StyledIcon
+                name={filterOption.open ? "chevronUp" : "chevronDown"}
+                size={"36"}
+              />
+            </Header>
+            {filterOption.open &&
+              filterOption.items.map((item, itemidx) => {
+                return (
+                  <FilterOption>
+                    <OptionGrid
+                      onClick={
+                        item.filterKey
+                          ? () => {
+                              updateFilterOption(item.filterKey)
+                            }
+                          : () => {
+                              setShowOptions(idx, itemidx, !item.showOptions)
+                            }
+                      }
+                    >
+                      <IconContainer>{item.icon}</IconContainer>
```

Comment:
The icon is presentational, we should hide it from SRs.

```suggestion
                      <IconContainer aria-hidden="true">{item.icon}</IconContainer>
```